src/gasman.c

```diff
@@ -114,6 +114,7 @@
 #include "gasman.h"
 #include "gasman_intern.h"
 
+#include "error.h"
```

Comment:
But if you include `error.h` now, why do we still need `TooLargeAllocError` declared in a separate source file?